views/routes/index.jsx

```diff
@@ -1,6 +1,7 @@
 import { Routes, Route, Navigate, BrowserRouter } from 'react-router-dom'
 
 import { routes } from './routes'
+import AdminCheck from '../hoc/AdminCheck'
```

Comment:
is HOC the right directory? This component is not a HOC.

Reply:
[68c51a1](https://github.com/AMP-SCZ/dpdash/pull/730/commits/68c51a17778cbfe24e7595789831fc3b9ca73a37)